src/Catty/PlayerEngine/Instructions/Sound/PlaySoundBrick+Instruction.swift

```diff
@@ -32,7 +32,7 @@
             let fileName = sound.fileName
             else { return .invalidInstruction }
 
-        let filePath = projectPath + kProjectSoundsDirName
+        let filePath = "\(projectPath + Util.defaultSceneName(forSceneNumber: 1))/\(kProjectSoundsDirName)"
```

Comment:
The sound does not necessarily be located in the first scene (also the first scene can have a different name).